packages/core/src/components/link-button/link-button.scss

```diff
@@ -0,0 +1,71 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Siemens AG
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@import 'common-variables';
+@import './components/buttons';
+
+:host {
+  display: inline-flex;
+  height: $x-large-space;
+  font-size: $font-size-default;
+  font-weight: $font-weight-normal;
+  min-width: 4.2rem;
```

Comment:
Value seems odd. Maybe setting it to something like $x-large-space would be better?